cmd/machine-controller/main.go

```diff
@@ -420,6 +421,11 @@ func startControllerViaLeaderElection(runOptions controllerRunOptions) error {
 			runOptions.parentCtxDone()
 			return
 		}
+		if err := nodecsrapprover.Add(mgr); err != nil {
+			klog.Errorf("failed to add NodeCSRApprover controller to manager: %v", err)
+			runOptions.parentCtxDone()
+			return
+		}
```

Comment:
Should we have a CLI flag that would control should the controller be enabled or not?

Reply:
@xmudrii probably yes, we need a flag to enable `NodeCSRApprover` and it should be disabled by default (to avoid conflict with kubermatic)